docs/setup/pick-right-solution.md

```diff
@@ -154,6 +158,7 @@ AppsCode.com         | Saltstack    | Debian | multi-support | [docs](https://ap
 KUBE2GO.io          |              | multi-support | multi-support | [docs](https://kube2go.io) | Commercial
 Madcore.Ai           | Jenkins DSL  | Ubuntu | flannel     | [docs](https://madcore.ai)                        | Community ([@madcore-ai](https://github.com/madcore-ai))
 Platform9        |              | multi-support | multi-support | [docs](https://platform9.com/managed-kubernetes/) | Commercial
+Kubermatic       |              | multi-support | multi-support | [docs](https://github.com/kubermatic/docs/) | Commercial
```

Comment:
Should we make all the list alphabetically ? I'm +1 on it.

Reply:
I think that would make the most sense. Would you like me to update the list to descending alphabetical order?

Reply:
@steveperry-53 WDYT?

Reply:
I don't have a strong feeling about it. But I think descending alphabetical order would be fine.